WordPress/Jetpack/Classes/NUX/New Landing Screen/Views/InfiniteScrollerView.swift

```diff
@@ -0,0 +1,114 @@
+import Foundation
```

Comment:
Nit: importing `Foundation` when also importing `UIKit` is redundant.

Reply:
Sounds good - fixed in c2a514b1f2352b746fe3d6a14ff23fa7d9f01c2c.

Reply:
Sounds good - fixed in c2a514b1f2352b746fe3d6a14ff23fa7d9f01c2c.